libs/core/mappers/pagination.mapper.ts

```diff
@@ -0,0 +1,22 @@
+import { PaginationDto } from '../dtos/pagination.dto';
+import { Pagination } from '../models/pagination';
+
+export namespace PaginationMapper {
+
+	/**
+	 * Maps dto to model.
+	 * @param pagination - Pagination dto.
+	 * @param mapper - Mapper function for results field.
+	 * @returns Pagination model.
+	 */
+	export function fromDto<I, O>(pagination: PaginationDto<I>, mapper: (dto: I) => O): Pagination<O> {
```

Comment:
Please use correct name for generic parameters
```suggestion
	export function fromDto<TDto, TModel>(pagination: PaginationDto<TDto>, mapper: (dto: TDto) => TModel): Pagination<TModel> {
```